config.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
       ref: Dune
       table_name: dune_sync_test_table
 
-  - name: Sync Parameterized Dune Query with Multiple Types to Postgres
+  - name: Sync Non-parameterized Dune Query with Multiple Types to Postgres
```

Comment:
Ubernit: Can we give these more compact names? Or maybe we can define the `Job.__repr__` to incorporate details about source, dest...

Locally I have changed these to:

- d2p-test-types
- p2d-test
- d2p-test-basic

Reply:
Let's handle all that in a separate PR